src/ripple/nodestore/impl/DatabaseShardImp.cpp

```diff
@@ -1215,11 +1239,59 @@ DatabaseShardImp::getDatabaseImportStatus() const
             currentShard[jss::storedSeqs] = shard->getStoredSeqs();
 
         ret[jss::currentShard] = currentShard;
+
+        if (haltDatabaseImport_)
+            ret[jss::message] = "Database import initiated...";
```

Comment:
This is shown if the import process is stopping, so the message should be something like "Database import stopping..."